tap_zoom/schemas/webinars.json

```diff
@@ -212,6 +403,12 @@
             "string"
           ]
         },
+        "alternative_host_update_polls": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
```

Comment:
`close_registration` seems to be a deprecated field [doc](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/webinar)